client/src/js/components/Auth/SignUp.jsx

```diff
@@ -235,8 +228,16 @@ class SignUp extends React.Component {
               </div>
 
             </div>
+            <div className="modal-footer mx-5 pt-3 mb-1">
+              <p className="font-small grey-text d-flex justify-content-end">
+              Have an account?
```

Comment:
`Have an account?` should be nested into the `<p>` tag and also check for other occurrence
Example
```
<p className="font-small grey-text d-flex justify-content-end">
    Have an account?
    <Link to="/signin" href="/signin" className="blue-text ml-1">
        Sign In
    </Link>
</p>
```
This makes it a bit easier to understand the code